docs/topics/Rate_Limits.md

```diff
@@ -50,15 +50,15 @@ Note that the normal rate-limiting headers will be sent in this response. The ra
 ```
 < HTTP/1.1 429 TOO MANY REQUESTS
 < Content-Type: application/json
-< Retry-After: 6457
+< Retry-After: 65
 < X-RateLimit-Limit: 10
 < X-RateLimit-Remaining: 0
 < X-RateLimit-Reset: 1470173023
 < X-RateLimit-Reset-After: 7
 < X-RateLimit-Bucket: abcd1234
 {
   "message": "You are being rate limited.",
-  "retry_after": 6457,
+  "retry_after": 64.57,
```

Comment:
These aren't equivalent. Did you mean 6.457?

Reply:
it's an example value. it doesn't need to be the exact same number

Reply:
would it not be sensible to give valid examples?

Reply:
If the rate limits have millisecond-precision I suggest also showcasing said precision in the example. Sure, this is a valid value, but just to be sure everyone gets that there can be up to three decimals I'd use `6.457` or `64.570`.

Reply:
I think you can safely imply millisecond precision is what the precision is in the response if it is a fractional value, unless you have access to a network that has latency of less than 1 millisecond to Discord.

Even in that case, rounding up to 1ms on a rate limit shouldn't harm you, unless you are excessively spamming the API to the point that this becomes an issue for you (which would possibly question what you are trying to do to it in the first place).